consensus/polybft/bitmap/bitmap_test.go

```diff
@@ -0,0 +1,56 @@
+package bitmap
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBitmap_Set(t *testing.T) {
+	data := []int{8, 15, 0, 7, 31, 60, 112, 7, 16, 241, 189, 60, 0, 19, 14, 25}
+	unique := map[int]struct{}{}
+
+	b := Bitmap{}
+
+	for _, v := range data {
+		unique[v] = struct{}{}
+
+		b.Set(uint64(v))
+	}
+
+	// check if only values from data are populated
+	for _, v := range data {
+		assert.True(t, b.IsSet(uint64(v)))
+	}
```

Comment:
How do you feel about moving this assertion in the first loop when you're setting the value? There is no need to iterate over the data again